### PR TITLE
Create 3D regression example

### LINKED ISSUE #1733: Create tutorial for 3d regression.
I'm always frustrated when deep learning libraries (or books on deep learning libraries) have an example for classification but not regression.  I realize that for the experienced user, it's easy to modify a classification example to regression, but for the inexperienced user, this can be difficult without a concrete example.

I will create a 3D regression tutorial very similar to the 3D classification tutorial. I will have to find a real-valued and continuous feature, and I will probably need to use a different publicly available data set.

A possible alternative would be to put commented code in the classification example, showing how to transform it into a regression example. These comments could be unwieldy and distracting, particularly if you use a different data set.

### PR DESCRIPTION
**Pull request by paul-reiners**
Fixes #1733.

### Description
Create 3D regression example.

### Checks
- [x] Avoid including large-size files in the PR.
- [x] Clean up long text outputs from code cells in the notebook.
- [x] For security purposes, please check the contents and remove any sensitive info such as user names and private key.
- [x] Ensure (1) hyperlinks and markdown anchors are working (2) use relative paths for tutorial repo files (3) put figure and graphs in the `./figure` folder
- [x] Notebook runs automatically `./runner.sh -t <path to .ipynb file>`

### CODE REVIEW

**Comment by paul-reiners** [comment]
Fixed warnings.

File: 3d_regression/densenet_training_array.ipynb [hunk 1]
```diff
@@ -0,0 +1,372 @@
+{
```
Comment (KumoLiu) [inline 3d_regression/densenet_training_array.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #2.    import math
</pre><p> </p><p>Please move the import into "setup import" section.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Project-MONAI/tutorials/pull/1737/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

**Comment by KumoLiu** [comment]
Thanks for the PR, overall looks good to me.

Could you please fix the DCO error as recommend: https://github.com/Project-MONAI/tutorials/pull/1737/checks?check_run_id=26606931703

File: 3d_regression/densenet_training_array.ipynb (hunk 1 above)
Comment (ericspod) [reply 3d_regression/densenet_training_array.ipynb:1]: <p>I would suggest instead to get rid of the <code>math</code> import and use Numpy only for that, so line 60 would be <code>rmse = np.sqrt(mse)</code>. The import  for <code>Regressor</code> should move to the <code>Setup Import</code> section.  </p>

**Review by ericspod — COMMENTED** [review]
It looks overall good to me but we need to sort out the CICD issues and address comments. Once that's done I can have a second look over it, but that should all be straightforward then we can merge.

File: 3d_regression/densenet_training_array.ipynb (hunk 1 above)
Comment (paul-reiners) [reply 3d_regression/densenet_training_array.ipynb:1]: This has been done.

**Comment by paul-reiners** [comment]
This is ready to go now.  I did have a regression earlier today, which I fixed.

**Review by ericspod — APPROVED** [review]

**Pull request merged**